hs_core/hydroshare/resource.py

```diff
@@ -525,7 +533,8 @@ def create_new_version_resource(ori_res, new_res, user):
         set_to_private = True
 
     # add files directly via irods backend file operation
-    utils.copy_resource_files_and_AVUs(ori_res.short_id, new_res.short_id, set_to_private)
+    utils.copy_resource_files_and_AVUs(ori_res.short_id, new_res.short_id, set_to_private,
```

Comment:
Is there any record in the resource metadata of the owner-of-record? Or is it just in the AVU? I would recommend having it in the metadata as well so that we can easily query it from Django when people ask what is being counted against their quota.